Source/WebCore/page/ElementTargetingController.cpp

```diff
@@ -358,6 +358,56 @@ static String parentRelativeSelectorRecursive(Element& element, ElementSelectorC
     return emptyString();
 }
 
+static String computeHasChildSelector(Element& element)
+{
+    static NeverDestroyed tagsToCheckForUniqueAttributes = [] {
+        MemoryCompactLookupOnlyRobinHoodHashSet<QualifiedName> names;
+        names.add(HTMLNames::aTag);
+        names.add(HTMLNames::imgTag);
+        names.add(HTMLNames::timeTag);
+        names.add(HTMLNames::pictureTag);
+        names.add(HTMLNames::videoTag);
+        names.add(HTMLNames::articleTag);
+        names.add(HTMLNames::audioTag);
+        names.add(HTMLNames::iframeTag);
+        names.add(HTMLNames::embedTag);
+        names.add(HTMLNames::sourceTag);
+        names.add(HTMLNames::formTag);
+        names.add(HTMLNames::inputTag);
+        names.add(HTMLNames::selectTag);
+        names.add(HTMLNames::buttonTag);
+        return names;
+    }();
```

Comment:
Nit: I think we can omit the lambda here with an initalizer list?

```suggestion
    static NeverDestroyed<MemoryCompactLookupOnlyRobinHoodHashSet<QualifiedName>> tagsToCheckForUniqueAttributes(std::initalizer_list<QualifiedName> {
        HTMLNames::aTag,
        HTMLNames::imgTag,
        HTMLNames::timeTag,
        HTMLNames::pictureTag,
        HTMLNames::videoTag,
        HTMLNames::articleTag,
        HTMLNames::audioTag,
        HTMLNames::iframeTag,
        HTMLNames::embedTag,
        HTMLNames::sourceTag,
        HTMLNames::formTag,
        HTMLNames::inputTag,
        HTMLNames::selectTag,
        HTMLNames::buttonTag,
    });
```

Reply:
Yep, this works! I'll use `initializer_list` here, and also in `computeTagAndAttributeSelector` which has a similar global hashset.